mysql-test/suite/rocksdb/t/commit_in_the_middle_ddl.test

```diff
@@ -0,0 +1,27 @@
+--source include/have_rocksdb.inc
```

Comment:
Test only needs MyRocks, not as default, properly uses ENGINE=ROCKSDB where necessary.